service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -78,4 +78,7 @@
     @NotNull(message = "You need to add details to children")
     @Valid
     private final List<@NotNull(message = "You need to add details to children")Element<Child>> children1;
+    private final String childrenMigrated;
```

Comment:
Please remove

Reply:
yeah anything to do with childrenMigrated or the old children should be gone. Do not add it back in.

Reply:
No, that should be removed.